serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -1,18 +1,18 @@
 // * xref:index.adoc[Home]
-* xref:release_notes.adoc[Release notes]
+//* xref:release_notes.adoc[Release notes]
```

Comment:
I agree. These comments are also confusing. If we need to remove them, it's easier to just remove/move to another page/module instead of leaving it commented.